docs/src/@primer/gatsby-theme-doctocat/live-code-scope.js

```diff
@@ -51,6 +52,7 @@ export default function resolveScope(metastring) {
     ...doctocatComponents,
     ...primerComponents,
     ...(metastring.includes('drafts') ? drafts : {}),
+    ...deprecated,
```

Comment:
Out of curiosity, why is deprecated different to drafts in that we aren't checking metadata like the preceding line?

Reply:
I think it needs to be conditional, otherwise deprecated components will override components with the same name and you will always see deprecated components in scope

```js
...
...primerComponents, // has Button, ActionMenu
...
...deprecated // also has Button, ActionMenu
```